src/AutorestSwift/View Models/KeyValueViewModel.swift

```diff
@@ -154,10 +160,9 @@ struct KeyValueViewModel: Comparable {
     /**
      Convert the type into String format in Swift
      */
-    private static func formatValueForType(type: AllSchemaTypes, value: String) -> String {
+    private static func formatValueForType(signatureParameter: ParameterType, value: String) -> String {
+        let type = signatureParameter.schema.type
```

Comment:
A "swify"er signature might look like:

```suggestion
    private static func formatValue(forSignatureParameter signatureParameter: ParameterType, value: String) -> String {
        let type = signatureParameter.schema.type
```